src/APIClient/index.tsx

```diff
@@ -57,6 +58,16 @@ class APIClient implements APIClientInterface {
                 }
             }
         );
+        this.clientWarningSubscription = Emitter.addListener(
+            EVENTS.WARNING,
+            (event: APIClientWarningEvent) => {
+                if (event.serverUrl === this.baseUrl) {
+                    Alert.alert("Warning", event.warning, [{ text: "OK" }], {
```

Comment:
Why display an alert here? I guess this should be fwd to the applicstion making use of the lib (aka the example app) and let it handle the UX as they see fit